### PR TITLE
Release cppo 1.6.2. Fixes tests.

### PR DESCRIPTION
**Pull request by mjambon**

### CODE REVIEW

File: packages/cppo/cppo.1.6.2/opam [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+build: [
+  ["jbuilder" "subst"] {pinned}
+  ["jbuilder" "build" "-p" name "-j" jobs]
+]
+build-test: [
+  ["jbuilder" "runtest" "-p" name]
+]
+
+depends: [
+  "jbuilder" {build & >= "1.0+beta10"}
```
Comment (rgrinberg) [inline packages/cppo/cppo.1.6.2/opam]: Note that the minimum version is beta17 for this version because we are using the new `diff` action.

**Comment by jpdeplaix** [comment]
Awesome. Thanks !

**Comment by jpdeplaix** [comment]
@mjambon It seems I missed some CI checks (or they weren't appearing).
Could you also fix ```cppo_ocamlbuild``` ?
```
#=== ERROR while installing cppo_ocamlbuild.1.6.0 =============================#
# opam-version 1.2.2
# os           linux
# command      jbuilder runtest -p cppo_ocamlbuild
# path         /home/travis/.opam/system/build/cppo_ocamlbuild.1.6.0
# compiler     system (4.02.3)
# exit-code    1
# env-file     /home/travis/.opam/system/build/cppo_ocamlbuild.1.6.0/cppo_ocamlbuild-7124-74f651.env
# stdout-file  /home/travis/.opam/system/build/cppo_ocamlbuild.1.6.0/cppo_ocamlbuild-7124-74f651.out
# stderr-file  /home/travis/.opam/system/build/cppo_ocamlbuild.1.6.0/cppo_ocamlbuild-7124-74f651.err
### stderr ###
# [...]
# -# 5
# +# 5 "loc.cppo"
#        "loc.cppo"   5  
# -# 5
# +# 5 "loc.cppo"
#    ))
#  
# -# 8
# +# 8 "loc.cppo"
#     "loc.cppo"   8   
```

**Comment by rgrinberg** [comment]
Jacques-Pascal Deplaix <notifications@github.com> writes:

> @mjambon It seems I missed some CI checks (or they weren't appearing).
> Could you also fix ```cppo_ocamlbuild``` ?

Seems like the issue is caused by cppo_ocamlbuild running the old tests
for cppo. We could introduce a constraint here, but it seems
unreasonable to do this just for the tests (there is no incompatibility
between the packages).

**Comment by mjambon** [comment]
Thanks for making the release. I didn't have the time yesterday.

**Comment by mjambon** [comment]
> Thanks for making the release. I didn't have the time yesterday.

um, never mind. I was confused.

**Pull request merged**